soap-connect/integration-tests/src/test/java/org/mule/it/soap/connect/SoapFootballExtensionArtifactFunctionalTestCase.java

```diff
@@ -25,14 +23,22 @@
   @Rule
   public DynamicPort laLigaPort = new DynamicPort("laLigaPort");
 
-  private HttpServer footballService = new HttpServer(footballPort.getNumber(), null, null, new FootballService());
-  private HttpServer laLigaService = new HttpServer(laLigaPort.getNumber(), null, null, new LaLigaService());
+  @Rule
+  public final ExternalProcess footballServiceServer =
+      new ExternalProcess("java", "-cp", System.getProperty("soapHttpServerClasspath"), "org.mule.service.soap.server.HttpServer",
```

Comment:
why can't you just start jetty programatically? Why does it have to be off process?

Reply:
I need this to be an independent JVM so it has a plain classloader. In the future this should be an independent docker image as @juandesi suggested.